docs/virtual_machines/windows_virtio_drivers.md

```diff
@@ -5,14 +5,10 @@ Microsoft Windows running in a fully virtualized guest.
 
 ## Do I need virtio drivers?
 
-Yes. Without the virtio drivers, you cannot use
-[paravirtualized](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_getting_started_guide/sec-virtualization_getting_started-products-virtualized-hardware-devices#sec-Virtualization_Getting_Started-Products-paravirtdevices)
```

Comment:
Having a link to some kind of definition for the term "paravirtualized" seems useful, some readers may not know what that means.
If that page is not good, maybe we can find something else?

Reply:
@jean-edouard Yes I agree. Do you think we should keep the same link?
I tried finding another link having good information but didn't found any...
Do you have any references ?

Reply:
[VirtIO](https://wiki.libvirt.org/page/Virtio)'s main landing page goes into the benefits of paravirtualized devices. It does not explicitly define the term paravirtualization, but describes the benefits well. Perhaps this link could be added under the paragraph as a "for more information about VirtIO and paravirtualization, see:"

There is also a [Wikipedia Article](https://en.wikipedia.org/wiki/Paravirtualization) on the concept, but it seems to cover history more than current usage.

Reply:
I have added https://wiki.libvirt.org/page/Virtio link